vsphere/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,455 @@
+          Example use cases:
+            - You have vSphere tags that might conflict with vSphere attributes or tags added by this integration or the
+              Datadog Agent
+            - You want a prefix to better recognise tags coming from vSphere
+          Examples with using `vtag_` prefix:
+          - vtag_os_type:windows            # where `os_type` is the category name and `windows` the tag name
+          - vtag_application_name:my_app    # where `application_name` is the category name and `my_app` the tag name
+          It's not recommend to use `vsphere_` as prefix, since it's already used by this integration
+          e.g. vsphere_host, vsphere_type, etc.
+        value:
+          type: string
+          default: "''"
+          example: ''
+      - name: collect_attributes 
+        required: false
+        description: |
+          If true, the vSphere attributes will be collected and submitted as Datadog tags.
+          The final tag sent to Datadog is composed of 'attribute_prefix' + vSphere attribute category as key and
+          the vSphere attribute value as value:
+            <ATTR_PREFIX><ATTR_CATEGORY>:<ATTR_VAL>
+          Examples:
+          - os_type:windows
+          - application_name:my_app
+        value:
+          type: boolean
+          default: false
+          example: false
+      - name: attributes_prefix
+        required: false
+        description: |
+          Custom attributes attached to vSphere resources will be prefixed with this prefix when collected.
+          Example use cases:
+            - You have vSphere attributes that might conflict with vSphere tags or tags added by this integration or the
+              Datadog Agent
+            - You want a prefix to better recognise attributes coming from vSphere
+          Examples with using `vattr_` prefix:
+          - vattr_os_type:windows            # where `os_type` is the attribute key and `windows` the attribute value
+          - vattr_application_name:my_app    # where `application_name` is the attribute key and `my_app` the attribute
+                                             # value
+          It's not recommend to use `vsphere_` as prefix, since it's already used by this integration
+          e.g. vsphere_host, vsphere_type, etc.
+        value:
+          type: string
+          default: "''"
+          example: ''
+      - name: use_guest_hostname
+        required: false
+        description: |
+          If true, the check will use the guest hostname for VMs instead of the VM name
+          This requires the VM to have VMware tools installed in it. If the guest hostname is
+          not available, it will fallback to the VM name.
+          Use this if you install the agent in VMs so that the hosts are not duplicated
+          in the web application UI.
+        value:
+          type: boolean
+          default: false
```

Comment:
General comment, you can remove `default` if the value is same as `example` to avoid duplication.